packages/opds-browser/src/components/mergeRootProps.ts

```diff
@@ -9,15 +9,17 @@ export function findBookInCollection(collection: CollectionData, bookUrl: string
   return allBooks.find(book => book.url === bookUrl);
 }
 
-export function mapStateToProps(state) {
+export function mapStateToProps(state, ownProps) {
```

Comment:
do you still need ownProps here since it can't come from the state? won't it just pass ownProps through normally?